### PR TITLE
New version: OpenJS.NodeJS version 21.4.0

### PR DESCRIPTION
**Pull request by coolplaylinbot**
###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/129246)

### CODE REVIEW

File: manifests/o/OpenJS/NodeJS/21.4.0/OpenJS.NodeJS.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+- nodevars
+- npm
+- npx
+ReleaseDate: 2023-12-05
+AppsAndFeaturesEntries:
+- UpgradeCode: '{47C07A3A-42EF-4213-A85D-8F5A59077C28}'
+ElevationRequirement: elevatesSelf
+Installers:
+- Architecture: x86
+  InstallerUrl: https://nodejs.org/dist/latest/node-v21.4.0-x86.msi
```
Comment (BrandonWanHuanSheng) [inline manifests/o/OpenJS/NodeJS/21.4.0/OpenJS.NodeJS.installer.yaml:24]:
```suggestion
  InstallerUrl: https://nodejs.org/dist/21.4.0/node-v21.4.0-x86.msi
```

File: manifests/o/OpenJS/NodeJS/21.4.0/OpenJS.NodeJS.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  InstallerSha256: 6C86DAD32EED707F7AEDFC7F986AC5A7DFA32B5D4474095DBA8B3FB01F38037E
+  ProductCode: '{15F3F0B2-BA80-47CE-BB2B-EE7CD70067AE}'
+- Architecture: x64
+  InstallerUrl: https://nodejs.org/dist/latest/node-v21.4.0-x64.msi
```
Comment (BrandonWanHuanSheng) [inline manifests/o/OpenJS/NodeJS/21.4.0/OpenJS.NodeJS.installer.yaml:28]:
```suggestion
  InstallerUrl: https://nodejs.org/dist/v21.4.0/node-v21.4.0-x64.msi

```

File: manifests/o/OpenJS/NodeJS/21.4.0/OpenJS.NodeJS.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+  InstallerSha256: 2F040B8180F01D97D03D8B4E44601DE22956947651F6724B38FDB7C4618E251E
+  ProductCode: '{5B523ED9-563D-4D87-BBE1-2A0C6D70CE1E}'
+- Architecture: arm64
+  InstallerUrl: https://nodejs.org/dist/latest/node-v21.4.0-arm64.msi
```
Comment (BrandonWanHuanSheng) [inline manifests/o/OpenJS/NodeJS/21.4.0/OpenJS.NodeJS.installer.yaml:32]:
```suggestion
  InstallerUrl: https://nodejs.org/dist/v21.4.0/node-v21.4.0-arm64.msi

```

**Review by BrandonWanHuanSheng — CHANGES_REQUESTED** [review]
All in the latest directory must need to be replaced with the v with app version. directory.
@CoolPlayLin

**Review by stephengillie — APPROVED** [review]

**Pull request merged**